lib/repl.js

```diff
@@ -1259,10 +1259,11 @@ function defineDefaultCommands(repl) {
   repl.defineCommand('help', {
     help: 'Show repl options',
     action: function() {
-      var self = this;
-      Object.keys(this.commands).sort().forEach(function(name) {
-        var cmd = self.commands[name];
-        self.outputStream.write(name + '\t' + (cmd.help || '') + '\n');
+      Object.keys(this.commands).sort().forEach((name) => {
+        const cmd = this.commands[name];
+        const spaces = ' '.repeat(name.length - 9);
```

Comment:
Yes, it will fail once we add a command with more than 10 characters. The longest right now is `editor`. Do you think that warrants a safeguard for future additions?